python-iml-manager.spec

```diff
@@ -226,27 +210,27 @@ mkdir -p $RPM_BUILD_ROOT%{_sysconfdir}/rabbitmq
 touch $RPM_BUILD_ROOT%{_sysconfdir}/nginx/conf.d/chroma-manager.conf
 mkdir -p $RPM_BUILD_ROOT%{_sysconfdir}/rabbitmq
 mkdir -p $RPM_BUILD_ROOT%{_sysconfdir}/grafana/provisioning/dashboards
-install -m 644 %{SOURCE18} $RPM_BUILD_ROOT%{_sysconfdir}/grafana/
-cp -r %{SOURCE19} $RPM_BUILD_ROOT%{manager_root}
+cp -r grafana $RPM_BUILD_ROOT%{manager_root}
+mv $RPM_BUILD_ROOT%{manager_root}/grafana/grafana-iml.ini $RPM_BUILD_ROOT%{_sysconfdir}/grafana/
 mv $RPM_BUILD_ROOT%{manager_root}/grafana/dashboards/iml-dashboards.yaml $RPM_BUILD_ROOT%{_sysconfdir}/grafana/provisioning/dashboards
-cp %{SOURCE16} $RPM_BUILD_ROOT%{_sysconfdir}/nginx/default.d/iml-manager-redirect.conf
-cp %{SOURCE17} $RPM_BUILD_ROOT%{_sysconfdir}/rabbitmq/rabbitmq-env.conf
-cp %{SOURCE1} $RPM_BUILD_ROOT%{_sysconfdir}/init.d/chroma-host-discover
+cp iml-manager-redirect.conf $RPM_BUILD_ROOT%{_sysconfdir}/nginx/default.d/iml-manager-redirect.conf
+cp rabbitmq-env.conf $RPM_BUILD_ROOT%{_sysconfdir}/rabbitmq/rabbitmq-env.conf
+cp chroma-host-discover-init.sh $RPM_BUILD_ROOT%{_sysconfdir}/init.d/chroma-host-discover
 mkdir -p $RPM_BUILD_ROOT%{_mandir}/man1
-install %{SOURCE3}.gz $RPM_BUILD_ROOT%{_mandir}/man1
-install -m 644 %{SOURCE2} $RPM_BUILD_ROOT%{_sysconfdir}/logrotate.d/chroma-manager
+install chroma-config.1.gz $RPM_BUILD_ROOT%{_mandir}/man1
+install -m 644 logrotate.cfg $RPM_BUILD_ROOT%{_sysconfdir}/logrotate.d/chroma-manager
 mkdir -p $RPM_BUILD_ROOT%{_unitdir}/
-install -m 644 %{SOURCE4} $RPM_BUILD_ROOT%{_unitdir}/
-install -m 644 %{SOURCE5} $RPM_BUILD_ROOT%{_unitdir}/
-install -m 644 %{SOURCE6} $RPM_BUILD_ROOT%{_unitdir}/
-install -m 644 %{SOURCE7} $RPM_BUILD_ROOT%{_unitdir}/
-install -m 644 %{SOURCE8} $RPM_BUILD_ROOT%{_unitdir}/
-install -m 644 %{SOURCE9} $RPM_BUILD_ROOT%{_unitdir}/
-install -m 644 %{SOURCE10} $RPM_BUILD_ROOT%{_unitdir}/
-install -m 644 %{SOURCE11} $RPM_BUILD_ROOT%{_unitdir}/
-install -m 644 %{SOURCE12} $RPM_BUILD_ROOT%{_unitdir}/
-install -m 644 %{SOURCE13} $RPM_BUILD_ROOT%{_unitdir}/
-install -m 644 %{SOURCE14} $RPM_BUILD_ROOT%{_unitdir}/
+install -m 644 iml-corosync.service $RPM_BUILD_ROOT%{_unitdir}/
+install -m 644 iml-gunicorn.service $RPM_BUILD_ROOT%{_unitdir}/
+install -m 644 iml-http-agent.service $RPM_BUILD_ROOT%{_unitdir}/
+install -m 644 iml-job-scheduler.service $RPM_BUILD_ROOT%{_unitdir}/
+install -m 644 iml-lustre-audit.service $RPM_BUILD_ROOT%{_unitdir}/
+install -m 644 iml-manager.target $RPM_BUILD_ROOT%{_unitdir}/
+install -m 644 iml-plugin-runner.service $RPM_BUILD_ROOT%{_unitdir}/
+install -m 644 iml-power-control.service $RPM_BUILD_ROOT%{_unitdir}/
+install -m 644 iml-settings-populator.service $RPM_BUILD_ROOT%{_unitdir}/
+install -m 644 iml-stats.service $RPM_BUILD_ROOT%{_unitdir}/
+install -m 644 iml-syslog.service $RPM_BUILD_ROOT%{_unitdir}/
```

Comment:
Can we install `*.service` instead of enumerating them?

Reply:
Done

Reply:
Just kidding. 

These files need to enumerated here (or in the Makefile task) so we don't pull in the rust .service files as well.

I did remove the enumeration from the `%files` block though.